### ISSUE TITLE
Tables with spaces in them autocomplete fields to the wrong table

### ISSUE THREAD

**Issue by jgwinner** [HeidiSQL/HeidiSQL#1718]
**Preconditions**

* HeidiSQL version: 12.1.0.6537
* Database type and version: MariaDB 10.6.10 (AWS RDS)
* OS: Windows for HeidiSQL. DB: AWS RDS - Probably Linux. 

**Describe the bug**

If a table, for example `tabItem Defaults` is used in the IDE SQL editor, the autocomplete field names seem to be taken from the truncated table, i.e `tabItem`

**To Reproduce**

1. Create two tables
`tabItem`
`tabItem Defaults` 

with different fields.

2. Open a SQL editor tab.
3. Type SELECT `'tabItem Defaults'` (use back ticks not single quotes - back slash back tick didn't seem to escape)
4. Type a period to get the field completion window.
5. The fields will be from `tabItem` not `tabItem Defaults`

Note the use of backticks, not single quotes, around the table name - the application (ERPNext) actually uses case sensitive table names (but not field names). 

**Screenshots or Crash reports**

No crash - just incorrect functioning.

**Comment by ansgarbecker** [comment]
Yes, that's a known issue - spaces in table names are not supported in the completion proposal.
